crates/hir_ty/src/traits/chalk.rs

```diff
@@ -413,30 +417,36 @@ pub(crate) fn trait_datum_query(
 }
 
 fn well_known_trait_from_lang_attr(name: &str) -> Option<WellKnownTrait> {
+    use chalk_solve::rust_ir::WellKnownTrait::*;
+
     Some(match name {
-        "sized" => WellKnownTrait::Sized,
-        "copy" => WellKnownTrait::Copy,
-        "clone" => WellKnownTrait::Clone,
-        "drop" => WellKnownTrait::Drop,
-        "fn_once" => WellKnownTrait::FnOnce,
-        "fn_mut" => WellKnownTrait::FnMut,
-        "fn" => WellKnownTrait::Fn,
-        "unsize" => WellKnownTrait::Unsize,
+        "sized" => Sized,
+        "copy" => Copy,
+        "clone" => Clone,
+        "drop" => Drop,
+        "fn_once" => FnOnce,
+        "fn_mut" => FnMut,
+        "fn" => Fn,
+        "unsize" => Unsize,
+        "coerce_unsized" => CoerceUnsized,
         _ => return None,
     })
 }
 
-fn lang_attr_from_well_known_trait(attr: WellKnownTrait) -> &'static str {
+const fn lang_attr_from_well_known_trait(attr: WellKnownTrait) -> &'static str {
+    use chalk_solve::rust_ir::WellKnownTrait::*;
```

Comment:
Can you leave these as before? We have [this page](https://rust-analyzer.github.io/metrics/) and this import will pollute the results (see https://github.com/rust-analyzer/rust-analyzer/issues/1165).

Reply:
Yup, we also have "avoid local enum imports" in the style guide: https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#import-style